addons/mrp_account/views/mrp_production_views.xml

```diff
@@ -8,10 +8,12 @@
                 <field name="show_valuation" invisible="1"/>
             </xpath>
             <xpath expr="//button[@name='action_view_mrp_production_sources']" position="after">
-                <button string="Valuation" type="object"
-                    name="action_view_stock_valuation_layers"
-                    class="oe_stat_button" icon="fa-dollar" groups="stock.group_stock_manager"
-                    attrs="{'invisible': [('show_valuation', '=', False)]}" />
+                <div groups="base.group_no_one">
+                    <button string="Valuation" type="object"
+                        name="action_view_stock_valuation_layers"
+                        class="oe_stat_button" icon="fa-dollar" groups="stock.group_stock_manager"
+                        attrs="{'invisible': [('show_valuation', '=', False)]}" />
+                </div>
```

Comment:
you could probably set the group_no_one directly on the groups_id of this ir.ui.view xpaht